native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/persist-native/reflect-config.json

```diff
@@ -0,0 +1,24 @@
+[
+{
+  "name":"com.mysql.cj.conf.url.SingleConnectionUrl",
```

Comment:
Could u check with @TharmiganK on how to handle configs used for test cases?

Reply:
Lets use `ballerinax.mysql-driver` module for testing. And if we add these configs to mysql driver module then we don't need to add the configs here